src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    status: 'Under Construction',
+  },
+  reducers: {
+    checkStatus: (state) => {
+      state.status = 'Under Construction';
+    },
+  },
+});
+
+export const { checkStatus } = createSlice.actions;
+
+export default createSlice.reducers;
```

Comment:
While you did an amazing job Implementing reducers and actions you disabled eslint and it is against maicroverse policy to disable linters because linters help us write clean code. So it would be nice if you enabled eslint in all files were it has been disabled I also noticed you disabled eslint in your Books.js file please kindly enable it.

Reply:
Resolved. Thanks for review 👍